rust/fory-core/src/serializer/list.rs

```diff
@@ -47,7 +47,6 @@ pub(super) fn get_primitive_type_id<T: Serializer>() -> TypeId {
         TypeId::U16 => TypeId::U16_ARRAY,
         TypeId::U32 => TypeId::U32_ARRAY,
         TypeId::U64 => TypeId::U64_ARRAY,
-        TypeId::USIZE => TypeId::USIZE_ARRAY,
```

Comment:
why we remove `USIZE` here? I thought we should add `USIZE/ISIZE` here too

Reply:
Fixed